### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by TNT-King** [larsiusprime/polymod#82] [General]
How can we actully use Polymod. I am really noob to coding and stuff so i dont understand this

**Comment by TNT-King** [comment]
Like do i need to download source code or anything different

**Comment by Cheemsandfriends** [comment]
You can download through git here or you can download through haxelib.

For downloading to git the command is `haxelib git polymod` and the link of polymod, which is https://github.com/larsiusprime/polymod
And for haxelib is `haxelib install polymod`
